tests/x11/kontact.pm

```diff
@@ -22,9 +22,19 @@ sub run() {
     # To disable it run at first time start
     x11_start_program("echo \"[General]\" >> ~/.kde4/share/config/kmail2rc");
     x11_start_program("echo \"first-start=false\" >> ~/.kde4/share/config/kmail2rc");
-    sleep 2;
+
+    # this is the workaround as same as above but against kontact5
+    # once we got all KF5-based kdepim merged, we can drop the KDE4 workaround
+    x11_start_program("echo \"[General]\" >> ~/.config/kmail2rc");
```

Comment:
yeah, I don't like it either. I spent too much time playing with this annoying behavior, sometimes A behind of B and mainwindow, sometimes B behind of A and mainwindow, etc. in the past :)